src/scicookie/{{cookiecutter.project_slug}}/build-system/setup.py

```diff
@@ -0,0 +1,21 @@
+from setuptools import setup  # isort:skip
+
+# Available at setup time due to pyproject.toml
+from pybind11.setup_helpers import Pybind11Extension  # isort:skip
+
+# Note:
+#   Sort input source files if you glob sources to ensure bit-for-bit
+#   reproducible builds (https://github.com/pybind/python_example/pull/53)
+
+ext_modules = [
+    Pybind11Extension(
+        "{{ cookiecutter.project_slug }}._core",
+        ["src/main.cpp"],
```

Comment:
You should take a look at the path here again.

Reply:
this is an interesting discussion .. but I think we can have this discussion after we merge this PR
I am also checking this discussion with Leah (pyOpenSci) to see if there is already something recommended by pyOpenSci